users/utils.py

```diff
@@ -7,3 +7,11 @@ def user_media_path(obj, filename):
 def artwork_media_path(obj, filename):
     """set artwork media path"""
     return f"artwork/{obj.artwork_id}/image/{filename}"
+
+
+def generateHandle(handle, count = 0):
```

Comment:
snake case

Reply:
@earvingemenez change the format from the camel case to snake case